package.json

```
@@ -16,11 +16,12 @@
   ],
   "license": "MIT",
   "main": "index.js",
+  "types": "index.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/fb55/bitfield"
   },
   "scripts": {
     "test": "standard && tape tests.js"
   }
-}
+}
```

```suggestion
}

```
My auto formatter seems to have omitted the newline at the end of the file. This might need to be re-added for consistency.